src/teleop/gui/src/components/AutonWaypointEditor.vue

```diff
@@ -561,48 +561,44 @@ export default {
         };
       });
 
-      // Always start the ids at 1 after the max id in currently in the array
-      let start_id =
-        this.storedWaypoints.length > 0
-          ? Math.max(...this.storedWaypoints.map((waypoint) => waypoint.id)) + 1
-          : 0;
-
+      // Start AR tag is always 0
       this.storedWaypoints.push({
         name: "Start",
-        id: start_id,
+        id: 0,
         lat: convertDMS(coordinates[0].lat, "D").d,
         lon: convertDMS(coordinates[0].lon, "D").d,
         gate: false,
         post: false,
       });
-      // Add Waypoints
+
+      // Add Waypoints, which we set as sentinel value -1.
       for (let i = 1; i < 4; ++i) {
         this.storedWaypoints.push({
           name: "Waypoint " + i,
-          id: start_id + i,
+          id: -1,
           lat: convertDMS(coordinates[i].lat, "D").d,
           lon: convertDMS(coordinates[i].lon, "D").d,
           gate: false,
           post: false,
         });
       }
 
-      // Add AR Tag Posts
+      // Add AR Tag Posts with IDs 1-3
       for (let i = 4; i < 7; ++i) {
```

Comment:
Now that the start_id logic is gone, I see no reason to leave this for loop starting at 4

Reply:
Hmm well then it lines up with coordinates. Either way we're gonna have to add or subtract within this loop. Will try to make this a little more readable though

Reply:
Now keeping track of things with two variables, tag_id and coord_num